From 3b55c0da6149e2977be7f95d092cfdddd7b232e6 Mon Sep 17 00:00:00 2001
From: Rong Bao <webmaster@csmantle.top>
Date: Wed, 26 Aug 2026 15:59:42 +0000
Subject: [PATCH] Bug 2065815 - Rename usage of macro Success to X11Success in
 nsXRemoteServer.cpp.  a=RyanVM

This is a fixup to Bug 2065007.

Original Revision: https://phabricator.services.mozilla.com/D321140

Differential Revision: https://phabricator.services.mozilla.com/D321524
---
 toolkit/components/remote/nsXRemoteServer.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/toolkit/components/remote/nsXRemoteServer.cpp b/toolkit/components/remote/nsXRemoteServer.cpp
index 297e573af82e..79a44b3ff3da 100644
--- a/toolkit/components/remote/nsXRemoteServer.cpp
+++ b/toolkit/components/remote/nsXRemoteServer.cpp
@@ -110,7 +110,7 @@ bool nsXRemoteServer::HandleNewProperty(XID aWindowId, Display* aDisplay,
 
     // Failed to get property off the window or
     // got a part only
-    if (result != Success || bytes_after != 0) {
+    if (result != X11Success || bytes_after != 0) {
       return false;
     }
 

